user_docs/en/changes.t2t

```diff
@@ -29,10 +29,20 @@ What's New in NVDA
 
 
 == Changes for Developers ==
+
+=== Warnings ===

```

Comment:
I don't think these can be clarified as warnings. I'd say they're just improvements.

Reply:
I agree that "Warning" does not clarify things at all.
"Improvements" is still better.
However are there cases where the change is not an improvement? If yes, a more neutral term would be more suitable.

Reply:
My intention here was more neutral definition of "warning" i.e. "advanced notices".
Maybe it would be better to ungroup these for now.

```suggestion
```